purchase_report_custom_hls/__manifest__.py

```diff
@@ -8,10 +8,11 @@
     "website": "https://www.quartile.co",
     "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```
For depending on purchase_order_secondary_unit.

Reply:
I edited it. Thank you.